src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Server/Controllers/WeatherForecastController.cs

```diff
@@ -36,9 +36,11 @@ public class WeatherForecastController : ControllerBase
         };
 
         private readonly ILogger<WeatherForecastController> _logger;
+#if (GenerateApiOrGraph || OrganizationalAuth || IndividualB2CAuth)
```

Comment:
Do we need GenerateApiOrGraph? even if the Web API does not call anything the scope needs to be verified?

Reply:
That's a valid question, though I'm not sure if I should be making that call. When I wrote the condition, I simply looked at where `scopeRequiredByApi` was used. In this case, I'm trying to accommodate the calls on line 58, 95 and 117. Are you saying that we should consider removing lines 58 and 95?

Reply:
No, we should not remove 58 and 95, @JunTaoLuo. It's just that 
`GenerateApiOrGraph` implies `OrganizationalAuth` or `IndividualB2CAuth` (you can only generate API in the case of  `OrganizationalAuth` or `IndividualB2CAuth` and you can only generate Graph in the case of  `OrganizationalAuth`), and therefore `GenerateApiOrGraph` in line 39 is redundant (but not harmful)

Reply:
Ah, I misunderstood what you said, and I see what you mean now. Yea, this can be simplified.